docs/sql/ddl.txt

```diff
@@ -812,3 +812,29 @@ node startup defined configuration file value or to its default value::
     +--------------------------------------------------------------------------+
     SELECT 1 row in set (... sec)
 
+
+Show Create Table
+=================
+
+To print a user-created table the ``SHOW CREATE TABLE`` statement can be used. This statement displays the DDL-schema of an existing table in the cluster. The following statement creates a table with ident ``my_table`` and defines it's schema. ::
+
+    cr> create table my_table (
+    ...   first_column integer,
+    ...   second_column string,
+    ...   third_column datetime,
+    ... );
+    CREATE OK (... sec)
+
+If the ``SHOW CREATE TABLE`` statement gets invoked, the schema of the previous created table gets returned. ::
+
+    cr> show create table my_table;
```

Comment:
please take a look at the `SqlFormatter` to see how the statement will be formatted